client/src/components/EditPost/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useContext } from 'react';
+import PropTypes from 'prop-types';
+
+import PostContext from '../../Context/PostContext';
+import { updatePost } from '../../API';
+import Post from '../Post';
+
+const EditPost = ({ postId }) => {
+  const { state } = useContext(PostContext);
+  const [isSaveLoading, setIsSaveLoading] = useState(false);
+  const [isPublishedLoading, setIsPublishedLoading] = useState(false);
+
+  const handlePublish = async () => {
+    setIsPublishedLoading(true);
+    const body = {
+      author: '5d5d60a51930b06b21e65765',
+      title: state.title,
+      content: state.content,
+      category: state.category,
+      public: state.isPublic,
+      published: true,
+    };
+    await updatePost(body, postId);
+    setIsPublishedLoading(false);
+  };
+
+  const handleSave = async () => {
+    setIsSaveLoading(true);
+    const body = {
+      author: '5d5d60a51930b06b21e65765',
```

Comment:
change this hardcoded  value after implementing login

Reply:
Yes. Will get these values from context.